api.swagger.yaml

```diff
@@ -897,6 +897,14 @@ paths:
     get:
       tags:
         - users
+      parameters:
+        - name: trimmed_detail
+          description: 'When true taxonomy details and the is_line_manager flag will not be present.'
```

Comment:
I'm probably being pedantic here but maybe it would be better longer term to say 'Returns a trimmed user schema.' rather than specific fields?